src/main/java/de/fearnixx/jeak/teamspeak/voice/connection/info/ConfigVoiceConnectionInformation.java

```diff
@@ -0,0 +1,98 @@
+package de.fearnixx.jeak.teamspeak.voice.connection.info;
+
+import com.github.manevolent.ts3j.identity.LocalIdentity;
+import de.mlessmann.confort.api.IConfig;
+import de.mlessmann.confort.api.IConfigNode;
+import de.mlessmann.confort.api.except.ParseException;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+public class ConfigVoiceConnectionInformation extends AbstractVoiceConnectionInformation {
+
+    private static final String NICKNAME_NODE = "nickname";
+    private static final String IDENTITY_NODE = "identity";
+    private IConfig configRef;
+
+    private String identifier;
+
+    public ConfigVoiceConnectionInformation(IConfig configRef, String identifier) {
+        this.identifier = identifier;
+        this.configRef = configRef;
+
+        try {
+            this.configRef.load();
+
+            if (this.configRef.getRoot() == null) {
+                this.configRef.createRoot();
+            }
+
+        } catch (IOException | ParseException e) {
+            throw new IllegalStateException("Could not read voice connection info config!");
```

Comment:
Please include the cause in the exception. The debug information will be swallowed otherwise.